tests/Comment/CommentTest.php

```diff
@@ -9,9 +9,7 @@
 use Sabberworm\CSS\Tests\ParserTest as TestsParserTest;
 
 /**
- * @covers \Sabberworm\CSS\Comment\Comment
- * @covers \Sabberworm\CSS\OutputFormat
- * @covers \Sabberworm\CSS\OutputFormatter
+ * @coversNothing
```

Comment:
Doesn't this cover `Comment::render()` and `ParserState::consumeWhitespace()`?

Reply:
`Comment::render()` is called. As far as I understand the tests (which is not far), they're about other elements rendering comments (or not) depending on the `OutputFormat` settings.

Reply:
Yes, it's `outputFormatter::comments()` where the controlling logic is.